workflow/Tiltfile

```diff
@@ -0,0 +1,79 @@
+load('ext://uibutton', 'cmd_button')
+
+def set_work_cmds(flow_name, res_name, work_cmds):
+    cmds = ['./write_workflow.sh', flow_name, res_name]
+    for i in work_cmds:
+        cmds.append(" ".join(i))
+    local(cmds)
```

Comment:
It might be nice to pass `echo_off=True` to any of these "internal" type commands to reduce output in logs